usr/src/test/test-runner/cmd/run.py

```diff
@@ -144,13 +145,16 @@ class Cmd(object):
     def __init__(self, pathname, outputdir=None, timeout=None, user=None):
         self.pathname = pathname
         self.outputdir = outputdir or 'BASEDIR'
-        self.timeout = timeout or 60
+        self.timeout = timeout
         self.user = user or ''
         self.killed = False
         self.result = Result()
 
+        if self.timeout == None:
+            self.timeout = 60
+
```

Comment:
I don't think checking for TypeError here is required. At this point, timeout can only be either None or an int, and if it's the former we change it to an int.